plugin-server/src/main/ingestion-queues/session-recording/process-event.ts

```diff
@@ -334,7 +335,9 @@ export const createSessionReplayEvent = (
         event_count: Math.trunc(events.length),
         message_count: 1,
         snapshot_source: snapshot_source || 'web',
+        // we can't default this one, since we now have multiple libraries in production
```

Comment:
We can infer the SDK from the `User-Agent` header, if $lib or User-Agent not available, we default to the Web SDK

Reply:
given the spread of SDKs now I'd much rather know for sure or default to none. for querying it's better to miss some on purpose than include some incorrectly. for billing 🙈

Reply:
makes sense.

Reply:
so we can read from $lib, if not available parse it from user-agent, otherwise `none`.
or vice versa, Mobile SDKs have user-agent and $lib, at least the most recent versions (a couple months back).